gematria/experiments/access_pattern_bm/contiguous_matrix.h

```diff
@@ -0,0 +1,14 @@
+#ifndef ACCESS_PATTERN_BM_CONTIGUOUS_MATRIX_H_
+#define ACCESS_PATTERN_BM_CONTIGUOUS_MATRIX_H_
+
+#include <iostream>
+
+namespace gematria {
+
+int *CreateRandomContiguousMatrix(const std::size_t size);
+void FlushContiguousMatrixFromCache(int *matrix, std::size_t size);
+void DeleteContiguousMatrix(int *matrix);
+
+}  // namespace gematria
+
+#endif
```

Comment:
_Read the other comments before acting on this one, if we merge all into one file, no action would be needed here._

The `endif` should have a comment that says this is for the header guard. Here:

```
#endif  // GEMATRIA_EXPERIMENTS_ACCESS_PATTERN_BM_CONTIGUOUS_MATRIX_H_
```

Same for the other headers.